discovery-aws-api/src/test/scala/akka/discovery/awsapi/ec2/Docs.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2017-2018 Lightbend Inc. <http://www.lightbend.com>
```

Comment:
```suggestion
 * Copyright (C) 2018 Lightbend Inc. <http://www.lightbend.com>
```

Reply:
@dhpiggott the header is auto-generated by sbt test:compile,  not sure why it's 2017-2018 and not just 2018, let's ask @raboof :-)

Reply:
I think currently we have the `headerPlugin` configured so that all files have the same copyright header.

We want all files to have at least the current year (see also #390).

Indeed it would also be fine to add only the current year to new files, but keep 2017 intact for existing files. I think we have that configured in Akka.